haskell-indexer-pipeline-ghckythe/src/Language/Haskell/Indexer/Pipeline/GhcKythe.hs

```diff
@@ -36,15 +37,23 @@ import Language.Haskell.Indexer.Frontend.Kythe (toKythe)
 
 import qualified Language.Kythe.Schema.Raw as Raw
 
+import System.IO
+
 ghcToKythe
     :: MVar () -> GhcArgs -> AnalysisOptions
     -> Raw.VName
-    -> ([Raw.Entry] -> IO ())
+    -> (Handle -> [Raw.Entry] -> IO ())
     -> IO ()
 ghcToKythe globalLock ghcArgs analysisOptions baseVName sink =
-    withTypechecked globalLock ghcArgs analysisOptions collect
-  where
-    collect xref = do
+    withTypechecked globalLock ghcArgs analysisOptions (collect (sink stdout) baseVName)
+
+
+pluginContinuation action h _ aoes baseVName sink
```

Comment:
needs typesig and/or explanation. Especially `aoes` :)

Reply:
Yes, you're right I have no idea now what the type of this is.